src/command/pss/send.ts

```diff
@@ -0,0 +1,81 @@
+import { readFileSync } from 'fs'
+import { LeafCommand, Option } from 'furious-commander'
+import { pickStamp } from '../../service/stamp'
+import { fileExists } from '../../utils'
+import { isHexString } from '../../utils/hex'
+import { PssCommand } from './pss-command'
+
+export class Send extends PssCommand implements LeafCommand {
+  public readonly name = 'send'
+
+  public readonly description = 'Send to recipient or target with Postal Service for Swarm'
```

Comment:
This description little bit confuses me. I can send message to a `target`, who is the `recipient` in the first interpretation for me. The meaning of `recepient` is somehow a public key here which changes how the message will be encrypted, but still the message will land on the `target` side. Moreover the `target` will be the mined message's prefix that will land at the _target node_ because of the kademlia routing.
So I would say here `Send a message to a target Bee node with Postage Service for Swarm`,
where the `target` is the target bee node's overlay address' prefix and the `recepient` is  the target Bee node's PSS public key to be more clear about these terms.